### PR TITLE
IMPROVE: Responsiveness

### LINKED ISSUE #2: Better Responsiveness across screens of different sizes
As an EmbedChat developer
**I Need to:** Make the EmbedChat component Responsive
**So That:** It fits best on screens of all sizes and gives users usage options based on screen size

**Acceptance Criteria**
- Emoji picker responsiveness.
- Defaults
  - width = 100%
  - height = 1/4 of the screen size
- Full-screen button on top right to expand the component to occupy the entire screen.
  - Show/hide features based on current size of the component.

### PR DESCRIPTION
**Pull request by sidmohanty11**
# Improving Responsiveness for both smaller and larger screens

## Acceptance Criteria fulfillment

- [x] minimize-maximize power
- [x] modifying height to 40% of the screen
- [x] emoji picker responsiveness

Fixes #2 

## Video/Screenshots

https://user-images.githubusercontent.com/73601258/174111578-735c4409-d3bb-4984-a46b-90eee81b8212.mp4

### CODE REVIEW

**Comment by sidmohanty11** [comment]
@RonLek is this the desired outcome (for minimize-maximize)? Or do we want to adjust the `width` accordingly too? Like 60% in desktop mode (minimized) and 100% when maximized or using a mobile screen.

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-import {ChatBody} from "./components/ChatBody/index";
-import {ChatHeader} from "./components/ChatHeader/index";
-import {ChatInput} from "./components/ChatInput/index";
+import { ChatBody, ChatHeader, ChatInput } from "./components";
 
 export const RCComponent = ({
   isClosable = false,
   setClosableState,
   moreOpts = false,
   width = "100%",
```
Comment (sidmohanty11) [inline src/index.js:10]: `width` defaults to 100%

**Comment by sidmohanty11** [comment]
@RonLek any pointers about how we should go about emoji picker responsiveness?
- Should we put a wrapper box around the `EmojiPicker` and try to shrink and grow it accordingly?

**Comment by sidmohanty11** [comment]
## Update

https://user-images.githubusercontent.com/73601258/174301496-ad33fd93-f169-495b-973f-d43b6458bd34.mp4

**Review by RonLek — APPROVED** [review]
LGTM! 💯

**Pull request merged**